db/external_sst_file_basic_test.cc

```diff
@@ -800,6 +800,7 @@ TEST_P(ExternalSSTFileBasicTest, IngestFileWithMultipleValueType) {
   bool verify_checksums_before_ingest = std::get<1>(GetParam());
   do {
     Options options = CurrentOptions();
+    options.disable_auto_compactions = true;
```

Comment:
[Note] With this PR, file ingestion results will depend on on-going compaction too. Disable auto ones so this test is remained untouched. Also auto-compaction is irrelevant to this test after inspection.